src/helpers/updateHistoryRecord.js

```diff
@@ -0,0 +1,15 @@
+import { getCurrentUrl } from '../helpers.js';
+
+const updateHistoryRecord = (url = null, customData = {}) => {
+	url = url || getCurrentUrl({ hash: true });
```

Comment:
Why did you add the optional hash here? Missed that before

Reply:
Sorry, wrong context... why does `getCurrentUrl()` now accept returning the hash as well?

Reply:
`getCurrentUrl` by default returns the url without a hash, but updating the history record should keep any current hash. I figured it's clearner to make it an option in `getCurrentUrl` than to combine `getCurrentUrl() + location.hash` or something. Should we rethink that? Not too late 🤡